src/core_plugins/kibana/public/dashboard/viewport/dashboard_viewport_provider.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { store } from '../../store';
+import { Provider } from 'react-redux';
+import { DashboardViewportContainer } from './dashboard_viewport_container';
+
+export function DashboardViewportProvider(props) {
```

Comment:
This is a total non-issue, and you certainly don't have to change this, but you can avoid the added return by defining simple SFC's as arrow functions. Just a quick tip.

```
export const DashboardViewportProvider = props => (
  <Provider store={store}>
    <DashboardViewportContainer {...props} />
  </Provider>
);
```